### ISSUE TITLE
Suggestion: Separate checked/unchecked text for checkboxes and radio buttons

### ISSUE THREAD

**Issue by ivnc** [nvaccess/nvda#8936]
### Steps to reproduce:
Navigate to a checkbox with a NVDA locale with different adjective terminations depending on the noun gender(Spanish or Galician, for example) and analize the output message.
### Actual behavior:
NVDA has only a string to define checked and unchecked states translation both for checkboxes and radio buttons, so NVDA reports "casilla de verificación marcado/sin marcar" (the two genders don't match unless asuming the word that the adjective accompanies to is a omitted masculine noun such as "elemento"-element or "objeto"-object which are masculine in Spanish).
### Expected behavior:
Gender matching. In case of Spanish: "Casilla de verificación marcada" and "botón de opción marcado" (marcada vs. marcado).
### System configuration:
N/A
### Other questions:

This suggestion was made by some Spanish-speaking users through the local community.

#### Does the issue still occur after restarting your PC?
N/A
#### Have you tried any other versions of NVDA?
Yes, same result.

**Comment by josephsl** [comment]
Hi,

Six years later...

Sounds like a localization issue unless it turns out to be an issue inside NVDA itself. Because localization strings can change from release to release, I'm in favor of closing this one unless folks point out this is a more widespread problem across languages.

Thanks.

**Comment by Adriani90** [comment]
Yes this affects multiple languages, romanian too for example. Reopening.
Please don‘t close without logic argumentation.
